src/components/modules/epics/ContributionsEpic.tsx

```diff
@@ -307,35 +307,36 @@ export const ContributionsEpicComponent: (
                 tracking={ophanTracking}
             />
 
-            {!isReminderActive && (
```

Comment:
I moved this show/hide logic inside of the component. This is so that we only ever send one view event to ophan.